src/helpers.ts

```diff
@@ -550,32 +562,28 @@ export async function getInstantMixItems(
  */
 export async function getItemsForPlayback(
     userId: string,
-    query: ItemQuery
+    query: ItemsApiGetItemsRequest
 ): Promise<BaseItemDtoQueryResult> {
-    query.UserId = userId;
-    query.Limit = query.Limit ?? 100;
-    query.Fields = requiredItemFields;
-    query.ExcludeLocationTypes = 'Virtual';
-
-    if (query.Ids && query.Ids.split(',').length == 1) {
-        const item = await JellyfinApi.authAjaxUser(
-            `Items/${query.Ids.split(',')[0]}`,
-            {
-                dataType: 'json',
-                type: 'GET'
-            }
-        );
+    const params: ItemsApiGetItemsRequest = {
+        excludeLocationTypes: ['Virtual'],
+        fields: requiredItemFields,
+        limit: query.limit ?? 100,
+        userId
```

Comment:
@nielsvanvelzen I'm guessing `userId` can be dropped here too?

Reply:
Yes, basically everywhere where is not required by the SDK it is optional. The only endpoints that still need it are admin related or for the profile picture URL.